lib/features/children/overview/pages/family_overview_page.dart

```diff
@@ -37,7 +37,8 @@ class FamilyOverviewPage extends StatelessWidget {
           appBar: AppBar(
             centerTitle: false,
             title: state is FamilyOverviewUpdatedState &&
-                    state.profiles.where((p) => p.type == 'Child').isEmpty
+                    state.profiles.where((p) => p.type == 'Child').isEmpty &&
```

Comment:
can we incapsulate this logic somewhere in FamilyOverviewUpdatedState class with simple and readable getters? like:
`  List<Profile> get children {
    return profiles.where((p) => p.type == 'Child').toList();
  }

  List<Profile> get adults {
    return profiles.where((p) => p.type == 'Parent').toList();
  }

  bool get hasChildren {
    return children.isEmpty;
  }

  bool get isAdultSingle {
    return adults.length == 1;
  }
`
Also, taking into account further scale on grandparents or even more, shall we use more generic type name like Adult instead of Parent?

Reply:
'Parent'/'Child' matches the BE type, I agree Adult would be better

Reply:
bool get hasChildren {
return children.is**Not**Empty;
}